src/scss/components/_form.scss

```diff
@@ -242,8 +242,9 @@ $floating-in-height: 3.25em !default;
 
 .control {
     .help.counter {
-        float: right;
+        float: none;
```

Comment:
Please remove it!

Reply:
Ok, done. Because the default value of the property is none?

Reply:
Sure

Reply:
Yes, but i'm checking and float: right attribute removes the top margin, so you should restore it